lib/reviewit/reviewit.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.license     = 'MIT'
 
   s.add_runtime_dependency 'trollop', '~> 2.0'
-  s.add_development_dependency 'awesome_print', '~> 1.2'
+  s.add_runtime_dependency 'awesome_print', '~> 1.2'
```

Comment:
I just removed this required in commit fab07ba917a2d156a8303103500afab94df046b2, we good now.